From 16a1338dafdc12c023e292bb1569906bc2244215 Mon Sep 17 00:00:00 2001
From: bahitov1998 <40343243+bahitov1998@users.noreply.github.com>
Date: Wed, 28 Oct 2020 14:10:08 +0400
Subject: [PATCH] 6

---
 Snake/Class1.cs | 22 +++++++++++++++++++---
 1 file changed, 19 insertions(+), 3 deletions(-)

diff --git a/Snake/Class1.cs b/Snake/Class1.cs
index 3cd3de4..d0a9d42 100644
--- a/Snake/Class1.cs
+++ b/Snake/Class1.cs
@@ -6,14 +6,30 @@ namespace Snake
     { 
         static void Main(string[] args)
         {
-            int x1 = 1;
+            Point p1 = new Point();
+            p1.x = 1;
+            p1.y = 3;
+            p1.sym = '*';
+            Draw(p1.x, p1.y, p1.sym);
+
+            /*int x1 = 1;
             int y1 = 3;
             char sym1 = '*';
 
-            Console.SetCursorPosition(x1, y1);
-            Console.Write(sym1);
+            Draw(x1, y1, sym1);*/
+
+            /*int x2 = 4;
+            int y2 = 5;
+            char sym2 = '#';
+
+            Draw(x2, y2, sym2);*/
 
             Console.ReadLine();
         }
+        static void Draw(int x, int y, char sym)
+        {
+            Console.SetCursorPosition(x, y);
+            Console.Write(sym);
+        }
     }
 }
-- 
2.28.0.windows.1

